### DISCUSSION TITLE
Google pubsub lite as source and sink

### DISCUSSION THREAD

**Discussion by mittal-jatin** [redpanda-data/connect#1531] [Ideas]
So, google have come up with pubsub lite for cost optimization and as we have adopted both benthos and pubsub lite in our org widely, we need a support for google pubsub lite as both source and sink in benthos.

**Comment by mihaitodor** [comment]
There's an issue opened for this already: #1440. Please upvote it and feel free to submit a PR if you can. Otherwise, we'll get to it eventually.
